lib/fastlane/plugin/wpmreleasetoolkit/helper/configure_helper.rb

```diff
@@ -99,11 +99,16 @@ def self.configure_file_is_behind_local
       end
 
       def self.configure_file_commits_behind_repo
-        # Get a sily number of revisions to ensure we don't miss any
+        # Get a large number of revisions to ensure we don't miss any
         result = `cd #{repository_path} && git --no-pager log -10000 --pretty=format:"%H" && echo`
```

Comment:
Not sure why the tool uses this kind of solution instead of using ad-hoc git commands like `git rev-list --count x..y` or similar…

Reply:
That's cool. I'll implement in the context of this PR because there's no rush to merge it as it.

Reply:
👍 In that case you might also want to take the occasion to look at the git Ruby gem (that I think we already use in other parts of the release-toolkit and is already one of our dependencies)

Reply:
>Not sure why the tool uses this kind of solution

Because the author (me) was a noob 😜

Reply:
> That's cool. I'll implement in the context of this PR because there's no rush to merge it as it.

@mokagio FYI I am about to make a new release of the toolkit today to ship some new features I want to use in WPAndroid/WCAndroid. Since you mentioned you'd implement the suggested change above in this PR rather than a separate one, I won't include this PR it in the upcoming release… except if you would like me to merge it as is now to include it and plan for addressing the change in a future subsequent one?